README.md

```diff
@@ -46,6 +46,25 @@ if you prefer to prebuild internal dependencies, add `--skip-internal` to the ab
 
 Most packages will require a .env file. `.env.example` files indicate the required variables per package.
 
+### Local database
+
+Set up PostgreSQL on your machine and create the database 'tupaia', with credentials matching those in
+the .env file.
+
+The project requires importing an initial database dump. Grab the latest database dump from dev server using
+
+```bash
+yarn dump-database ~/path/to/key.pem
+```
+
+(pem key can be obtained in LastPass - Tupaia Main Server). This will create a db dump and pull it to your current directory. You also need to be added into the Security Groups of the server.
```

Comment:
I was under the impression that we removed all references to lastpass, is that not true?

Reply:
That's correct, although in some cases it's not possible without a further restructure of either the documentation or the code. There is a task for that, see https://github.com/beyondessential/tupaia-backlog/issues/1202

In this specific case, we could avoid those references by moving this documentation to a Tupaia resource, eg our BookStack. It would be easy to cover this process in a "Step 5" of the new [Tupaia Monorepo setup guide](https://docs.beyondessential.com.au/books/software-development/page/tupaia-monorepo-setup), but those scripts will still exist in the codebase, just be undocumented/non operational for people that don't have access to our server.

Thoughts?

Reply:
I think that's a great idea! Perhaps also add a comment at the start of the script itself mentioning that as well, in case anyone stumbles across it.

Reply:
I'd vote for keeping some documentation in place, and just have flags that mark the pathway for BES Internal, or Open Source Community within the readme. Personally, I don't see it as that bad having a reference to Lastpass in the readme, so long as there's a clear alternative for someone external to follow (in this case, it might be running refresh database using testDataDump). If anything, it makes things more transparent, and explains otherwise unanswered questions, like "what's this dump-database script about". 

Also happy for the BES Internal pathway to be a link to bookstack if others prefer that (and it would make more sense if it's a long set of steps or contains secrets), but again I still think that should be kept in the readme and flagged as BES Internal.

What do you guys think?

Reply:
I believe there are merits in both approaches. I'd say we defer this decision for when https://github.com/beyondessential/tupaia-backlog/issues/1202 is implemented. Even if we remove this reference to LastPass, we only "fake" more open source compatibility - people will still have a hard time figuring out how to setup the whole thing.

So let's keep this here for now, and discuss a more generic strategy of making our project approachable (eg exposing a docker container for local dev etc) when the time comes.

Reply:
I guess that we can still replace this with a link to our BookStack and an "BES internal" label? I wouldn't worry about the script because as @edmofro pointed out, it does work as long as a dev has a DB dump, which may obtained from a different source than our servers.